server.js

```diff
@@ -6,12 +8,11 @@
 
 const { createServer } = require('http')
 const { parse } = require('url')
-const _ = require('force-ssl-heroku')
```

Comment:
why removing this?

Reply:
should we remove it from dependencies?

Reply:
It was not used, yes we should probably remove it from dependencies as well.

Reply:
yep, I see it's a leftover before an explicit 301 was introduced